### DISCUSSION TITLE
AlmaLinux is out of GNS3?

### DISCUSSION THREAD

**Discussion by Ilike-IT** [GNS3/ubridge#93] [Q&A] [answered]
Hi everybody,
I've tried to follow your instructions on installation to Linux, unsuccessfully:
c -Wall -DLINUX_RAW   -c -o src/ubridge.o src/ubridge.c
In file included from src/ubridge.c:34:
src/ubridge.h:33:10: fatal error: pcap.h: No such file or directory
   33 | #include <pcap.h>
      |          ^~~~~~~~
compilation terminated.
make: *** [<builtin>: src/ubridge.o] Error 1
cc -Wall -DLINUX_RAW   -c -o src/ubridge.o src/ubridge.c
In file included from src/ubridge.c:34:
src/ubridge.h:33:10: fatal error: pcap.h: No such file or directory
   33 | #include <pcap.h>
      |          ^~~~~~~~
compilation terminated.

The 'libcap-devel' is installed and "pcap.h: No such file  " ! 
How to resolve the problem, could you assist me ?

**Comment by grossmj** [comment, marked as answer]
What instructions you are trying to follow?

**Reply by Ilike-IT** [reply]
Thanks for your message!
How can I install gns3 on Almalinux successfully? I did not find the instructions to do it.
